src/actions/actionToggleZenMode.tsx

```diff
@@ -1,12 +1,10 @@
 import { CODES, KEYS } from "../keys";
 import { register } from "./register";
-import { trackEvent } from "../analytics";
 
 export const actionToggleZenMode = register({
   name: "zenMode",
+  trackEvent: { category: "canvas" },
   perform(elements, appState) {
-    trackEvent("view", "mode", "zen");
-
     return {
```

Comment:
I've changed the event category/action/label here and also for videMode and gridMode, which means we'll lose continuity with past data. IMO ok.